sycl/include/CL/sycl.hpp

```diff
@@ -60,6 +60,7 @@
 #if SYCL_EXT_ONEAPI_BACKEND_LEVEL_ZERO
 #include <sycl/ext/oneapi/backend/level_zero.hpp>
 #endif
+#include <sycl/ext/oneapi/barrier.hpp>
```

Comment:
Should this be sycl/ext/oneapi/experimental/cuda/barrier.hpp instead?

Reply:
Yes